tests/integration/clusterProxyConfig_test.go

```diff
@@ -26,6 +28,36 @@ func (m *IntegrationSuite) TestClusterProxyConfig() {
 	// Creating the first CPC should succeed
 	err = client.Create(ctx, validCreateObj.Namespace, validCreateObj, result, metav1.CreateOptions{})
 	m.NoError(err, "Error returned during the creation of a valid clusterProxyConfig")
+
+	// Verify the API knows about the first object before we try to create a second one.
+	// Wait 2 minutes for the object to be created (should be more than ample time)
```

Comment:
nit: the comment states 2 minutes, but the actual code waits for 5
second nit: 5 minutes is a long time. I kinda prefer that to the 30+ minutes it takes to restart the entire testing suite, but is there no better way? could we poll for the API to be aware of the object in a loop with smaller sleeps or something?

Reply:
At least I added math to the code to make it clearer how long we're willing to wait.